components/salesforce_rest_api/salesforce_rest_api.app.mjs

```diff
@@ -23,11 +23,7 @@ export default {
       type: "string",
       label: "SObject Type",
       description: "Standard object type of the record to get field values from",
-      async options(context) {
-        const { page } = context;
-        if (page !== 0) {
-          return [];
-        }
```

Comment:
Why are we removing the pagination logic?

Reply:
I think that without this guard, the user can click the *Load More* button and will get the same list a second time, producing a long list with lots of duplicates.

Reply:
I think the UI handles the duplicates, I haven't had this issue and also just teste and it doesn't duplicate the list